From 496f51bccd23baaf2495423bca8d009b8bc22d32 Mon Sep 17 00:00:00 2001
From: Torsten Coym <torsten.coym@novaled.com>
Date: Thu, 26 Apr 2018 13:15:08 +0200
Subject: [PATCH] include ldms devicetree in Makefile

---
 arch/arm/boot/dts/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index a573819..e5f6cf8f 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -355,6 +355,8 @@ dtb-$(CONFIG_SOC_LS1021A) += \
 	ls1021a-qds.dtb \
 	ls1021a-twr.dtb
 dtb-$(CONFIG_SOC_VF610) += \
+	vf500-colibri-ldms.dtb \
+	vf610-colibri-ldms.dtb \
 	vf500-colibri-eval-v3.dtb \
 	vf610-colibri-eval-v3.dtb \
 	vf500-colibri-aster.dtb \
-- 
2.1.4

